bot/exts/utilities/realpython.py

```diff
@@ -32,13 +32,16 @@ def __init__(self, bot: Bot):
 
     @commands.command(aliases=["rp"])
     @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
-    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str) -> None:
+    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str=None) -> None:
         """
         Send some articles from RealPython that match the search terms.
-
         By default the top 5 matches are sent, this can be overwritten to
         a number between 1 and 5 by specifying an amount before the search query.
         """
+        if (user_search is None):
+            await ctx.send(HOME_URL)
```

Comment:
It would be nice to send this as an embed just for the nicer formatter. This also means you can specify the `url` kwarg so people can click on the embed title to open the page automatically.

Something simple like
```py
await ctx.send(embed=discord.Embed(title="Real Python Home Page", url=HOME_URL))
```

Reply:
This is what I had in mind for the original fix as well, having the embed makes it look 'clean'.